csharp/src/Drivers/Apache/Spark/SparkConnection.cs

```diff
@@ -50,6 +50,8 @@ public class SparkConnection : HiveServer2Connection
         const string InfoDriverVersion = "1.0.0";
         const string InfoVendorName = "Spark";
         const string InfoDriverArrowVersion = "1.0.0";
+        const int DecimalPrecisionDefault = 10;
```

Comment:
It'd be good to mention where these numbers came from.

Reply:
Will document in a future PR. Value comes from Databricks Spark documentation for the DECIMAL type.
https://docs.databricks.com/en/sql/language-manual/data-types/decimal-type.html#syntax